src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/MiningAndroidListener.java

```diff
@@ -37,6 +41,10 @@ public void onAndroidMine(AndroidMineEvent e) {
             slimefunItem.callItemHandler(BlockBreakHandler.class, handler -> {
                 if (handler.isAndroidAllowed(e.getBlock())) {
                     handler.onAndroidBreak(e);
+
+                    Collection<ItemStack> drops = e.getDrops();
+                    drops.clear();
+                    drops.addAll(slimefunItem.getDrops());
```

Comment:
The drop list should be updated *before* the event is passed onto the itemhandler.

Reply:
Then we get the same performance problem with blockstorage